api/tests/opentrons/protocol_engine/state/test_command_view.py

```diff
@@ -0,0 +1,126 @@
+"""Labware state store tests."""
+from collections import OrderedDict
+from pydantic import BaseModel
+from typing import Sequence, Tuple, cast
+
+from opentrons.protocol_engine import commands as cmd
+from opentrons.protocol_engine.state.commands import CommandState, CommandView
+
+
+from .command_fixtures import (
+    create_pending_command,
+    create_running_command,
+    create_failed_command,
+    create_completed_command,
+)
+
+
+def get_command_view(
+    commands_by_id: Sequence[Tuple[str, cmd.CommandType]] = ()
+) -> CommandView:
+    """Get a command view test subject."""
+    state = CommandState(commands_by_id=OrderedDict(commands_by_id))
+
+    return CommandView(state=state)
+
+
+def test_get_command_bad_id() -> None:
+    """get_labware_data_by_id should raise if labware ID doesn't exist."""
+    subject = get_command_view()
+
+    result = subject.get_command_by_id("asdfghjkl")
```

Comment:
Maybe instead or also:

```suggestion
    subject = get_command_view(
        { "correct-command-id": cast(cmd.CommandType, create_completed_command(BaseModel(), BaseModel())) }
    )

    result = subject.get_command_by_id("incorrect-command-id")
```

(This `cast()` might be unnecessary. See GitHub comment below.)

Reply:
Simplified casting and incorporated this change directly